### PR TITLE
GCMD command changes

### LINKED ISSUE #1905: Gcmd command blacklist
Like the wildcard command I think it would be a good idea to add a command blacklist. I.e. User x tries to gcmd player to doom themself would be blocked because of /doom being on the blacklist. Also, gcmding multiple users, e.g /gcmd gupta gcmd gupra2 say hi should be blocked.

### PR DESCRIPTION
**Pull request by ghost**
Added GCMD blacklist and blocked from console, resolves ~~#1905 #1899 and #1900~~ Issues superseded by #1906

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_gcmd.java [hunk 1]
```diff
@@ -1,56 +1,83 @@
...
             {
-                msg("Unknown error sending command.");
+                msg("Error sending command: " + ex.getMessage());
             }
         }
-        catch (Throwable ex)
+        catch (Exception e)
         {
-            msg("Error sending command: " + ex.getMessage());
+            msg("/gcmd <fromname> <outcommand>");
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_gcmd.java]: Just use return false

**Comment by Commodore64x** [comment]
Have added in message for GCMDing players, resolves #1900

**Comment by ghost** [comment]
Signed the CLA.

**Comment by ghost** [comment]
Testing for bugs now :P

**Comment by ghost** [comment]
I think some styling could be added to the message that alerts that a command has been run by you, any suggestions?

**Comment by ghost** [comment]
Fixed: Sender coming up as CraftPlayer
Fixed: Command not being blocked

**Comment by ghost** [comment]
I am pretty satisfied with this at the moment, if some other people want to test, go right ahead.

**Comment by ghost** [comment]
Lemon please remove the Needs CLA tag.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_gcmd.java (hunk 1 above)
Comment (lemonsked) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_gcmd.java]: Stil. needs this @Commodore64x

**Comment by mayokek** [comment]
I thought you can only create a PR per accepted issues.

**Comment by ghost** [comment]
@AvalancheYT at this point, the things that this PR features are very unlikely to get rejected. They will definitely help with gcmd command abuse by certain admins.

**Comment by Wild1145** [comment]
This PR will remain on hold until the associated issue is reviewed, discussed and approved.

Please also try to keep it to one PR to one Issue, not one PR for multiple issues.

**Review by Wild1145 — COMMENTED** [review]
I can see no file changes... At all... You appear to have 7 commits that do nothing...

**Comment by mayokek** [comment]
@wild1145, he have deleted the changes.

**Comment by ghost** [comment]
What??? I haven't deleted anything... Let me check @Wild1145

**Comment by ghost** [comment]
Superseeded by #1911

**Pull request closed without merging**